src/doubletdetection/plot.py

```diff
@@ -0,0 +1,105 @@
+from .doubletdetection import normalize_counts
+from sklearn.decomposition import PCA
+from MulticoreTSNE import MulticoreTSNE as TSNE
+import phenograph
+
+import os
+import warnings
+import numpy as np
+
+import matplotlib
+from matplotlib import font_manager
+try:
+    os.environ['DISPLAY']
+except KeyError:
+    matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+with warnings.catch_warnings():
```

Comment:
You can get rid of these font requests, too. They're some custom google fonts I used for my dissertation.